puppet/modules/jenkins_job_builder/files/theforeman.org/unmanaged_jobs.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'net/http'
+require 'json'
+
+def managed?(job)
+  job['description']&.include?('<!-- Managed by Jenkins Job Builder -->')
```

Comment:
What's the `&` in there? This may just be a lack of ruby knowledge from my side :)